graylog2-web-interface/src/views/components/searchbar/saved-search/SavedSearchForm.tsx

```diff
@@ -56,6 +57,8 @@ const SavedSearchForm = (props: Props) => {
   } = props;
   const disableSaveAs = !value || value === '' || disableCreateNew;
   const createNewTitle = isCreateNew ? 'Create new' : 'Save as';
+  const pluggableSaveViewControlFns = usePluginEntities('views.components.saveViewForm');
+  const pluggableSaveViewControls = pluggableSaveViewControlFns.map((controlFn) => controlFn()).filter((control) => !!control);
```

Comment:
This looks like a pattern that is repeated a couple of times. Maybe we can turn it into a custom hook (`useSaveViewFormControls`?) that does the `map`/`filter` step? This way we can also remove a bit of work from `executePluggableSearchDuplicationHandler` etc.

Reply:
Good idea.